app/views/staff/events/_proposal_tags.html.haml

```diff
@@ -1,8 +1,8 @@
 #show-proposal-tags
-  - if event.valid_proposal_tags.empty? && current_user.organizer_for_event?(event)
+  - if !event.public_tags? && current_user.organizer_for_event?(event)
```

Comment:
Has to check or the else doesn't get called. Its a 'add' or 'edit' proposal tags for the organizer view.